keps/sig-node/127-user-namespaces/README.md

```diff
@@ -670,6 +672,26 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+#### Kubelet and Kube-apiserver skew
+
+The apiserver and kubelet feature gate enablement are work fine in any combination:
+
+1. If the apiserver has the feature gate enabled and the kubelet doesn't, then pod will show that
+   field and the kubelet will ignore it. Then, the pod is created without user namespaces.
+1. If the apiserver has the feature gate disabled and the kubelet enabled, the pod won't show this
```

Comment:
should this be "2."?

Reply:
Both work. The "trick" is that using always one, you can re-arrange and never care about it. Markdown, when rendered, shows the proper number :)